manifest/v1alpha/slo/validation.go

```diff
@@ -138,6 +235,49 @@ var specCompositeValidationRule = validation.NewSingleRule(func(s Spec) error {
 	return nil
 })
 
+// CompositeSpec objective may only occur once, and may not be mixed with CountMetricsSpec or RawMetricSpec
+var specCompositeObjectiveValidationRule = validation.NewSingleRule(func(s Spec) error {
```

Comment:
why not include that rule in the `specMetricsValidation`? There's an exclusive condition for both raw and count metrics fields and composite and I think it should be placed within a single rule

Reply:
Removed, other rules check that.